simulation/simulation.py

```diff
@@ -14,15 +18,25 @@
 class SwarmBallSimulation(object):
     def __init__(self):
         # main simulation parameters
-        self.thresholds_positions_x = [200, 500, 800]
-        self.number_of_agents_per_thresholds = 10
+        self.number_of_clusters = 3
+        self.number_of_agents_per_cluster = 10
         self.goal_object_frame_dim = [100, 100]
+        self.enemy_speed = 0.5
         self.debug_mode = False
 
+        self._thresholds = []
+        self._map_segments = [
+            ((0, 100), (100, 100)),
+            ((100, 100), (200, 200)),
+            ((200, 200), (1280, 300))
+        ]
```

Comment:
@TheMoira tego formatu potrzebuje <3 jakby sie dalo od razu taki to by bylo supi